AdyenCheckout/CheckoutComponentBuilder.swift

```diff
@@ -15,14 +15,12 @@
     @_spi(AdyenInternal) import AdyenActions
 #endif
 
-public protocol AdyenCheckoutComponent: PresentableComponent {}
-
 internal enum CheckoutComponentBuilder {
     
     internal static func build(
         for paymentMethod: PaymentMethod,
         configuration: CheckoutConfiguration
-    ) throws -> AdyenCheckoutComponent {
+    ) -> PaymentComponent {
```

Comment:
I would argue that `throws` is a nice addition for ApplePay and other SDK-based components.